src/wp-admin/includes/deprecated.php

```diff
@@ -1588,3 +1588,29 @@ function image_attachment_fields_to_save( $post, $attachment ) {
 
 	return $post;
 }
+
+/**
+ * Queues comments for metadata lazy-loading.
+ *
+ * @since 4.5.0
+ * @since 6.3.0 Use wp_lazyload_comment_meta() for lazy-loading of comment meta.
```

Comment:
```suggestion
```

This isn't needed as it's covered by the `@deprecated` annotation. It was added to the function in an earlier commit as a few contributors were discussing whether this was suitable to deprecate :)